native_client/python/client.py

```diff
@@ -37,20 +37,19 @@
 N_CONTEXT = 9
 
 def convert_samplerate(audio_path):
-    sox_cmd = 'sox --norm {} -b 16 -t wav - channels 1 rate 16000'.format(audio_path)
+    sox_cmd = 'sox {} --type raw --bits 16 --channels 1 --rate 16000 - '.format(audio_path)
```

Comment:
Force raw in-par with NodeJS processing, and removes the need for `split('data')`